viewing_party/party.py

```diff
@@ -1,23 +1,152 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title == None or genre == None or rating == None:
+        new_movie = None
+    else:
+        new_movie = {"title": title, "genre": genre, "rating": rating}
+    return new_movie
 
+def add_to_watched(user_data, movie):
+    watched_list = user_data["watched"]
+    watched_list.append(movie)
+    user_data["watched"] = watched_list  
+        
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    watchlist = user_data["watchlist"]
+    watchlist.append(movie)
+    user_data["watchlist"] = watchlist  
+
+    return user_data
+
+def watch_movie(user_data, movie):
+    # look up movie in watchlist
+    tracker_dict = None
+    for movie_dict in user_data["watchlist"]:
+        if movie == movie_dict["title"]:
+            add_to_watched(user_data, movie_dict)

```

Comment:
Great decision to use the `add_to_watched` function here! A lot of people didn't think to do this, but I think it's ideal code to reuse it.